yarn.lock

```diff
@@ -26,6 +26,13 @@
   dependencies:
     "@babel/highlight" "^7.0.0"
 
+"@babel/code-frame@^7.5.5":
```

Comment:
Why has the yarn.lock file changed if none of the dependencies have changed?

Reply:
I think this is from the prior commit. Just needs to be updated with another run of yarn.